ldclient/impl/events/event_context_formatter.py

```diff
@@ -17,24 +17,25 @@ def __init__(self, all_attributes_private: bool, private_attributes: List[str]):
             if ar.valid:
                 self._private_attributes.append(ar)
 
-    def format_context(self, context: Context) -> dict:
```

Comment:
Same comment on other PRs re: adding a named function, not disturbing existing

Reply:
Also done.